Please add the dotnet-as-script plugin

ariel-lenis opened the pull request:
<!-- This PR template only applies to permission changes. Ignore it for changes to the tool updating permissions in Artifactory -->

# Description

Hi, the additional information of this plugin is in the repo: https://github.com/jenkinsci/dotnet-as-script-plugin

# Permissions pull request checklist

<!--
Make sure to implement all relevant entries (see section headers to when they apply) and mark them as checked (by replacing the space between brackets with an "x"). Remove sections that don't apply, e.g. the second and third when adding a new uploader to an existing permissions file.
-->

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

https://issues.jenkins-ci.org/browse/INFRA-1162
https://issues.jenkins-ci.org/browse/HOSTING-320

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

daniel-beck commented on permissions/plugin-dotnet-as-script.yml:
```diff
@@ -0,0 +1,6 @@
+---
+name: "dotnet-as-script"
+paths:
+- "org/jenkins-ci/plugins/dotnet-as-script"
```
Wrong path due to https://github.com/jenkinsci/dotnet-as-script-plugin/blob/805a9e36bf77cdeb32648d9c98f45a39b31278f7/pom.xml#L12

Please see this repo's readme how to map from groupId/artifactId to path.

daniel-beck commented:
Also, dotnetscript.com is a registered domain. If you're not the owner, you shouldn't use it as namespace.

ariel-lenis commented:
hi @daniel-beck sorry for the problems
This are the new changes:

- Fixed groupId from dotnetscript.com to org.jenkins-ci.plugins
- Renamed all java package names from com.dotnetscript to hudson.plugins.dotnetasscript
- Fixed some additional strings

Thanks!

daniel-beck commented:
> hudson.plugins.dotnetasscript

Doesn't really matter, but the `hudson` package name now only exists for historical reasons. New plugins now use `org.jenkinsci.plugins.` or `io.jenkins.plugins.`.
